tests/api/Jalasoft.TeamUp.Projects.API.Tests/PostProjectsTest.cs

```diff
@@ -31,5 +32,14 @@ public async void PostProject_Returns_CreateProject_Project()
             var okObjectResult = Assert.IsType<CreatedResult>(response);
             Assert.IsType<Project>(okObjectResult.Value);
         }
+
+        [Fact]
+        public async void PostProject_Returns_BadRequest()
+        {
+            var request = this.mockHttpContext.Request;
+            this.mockProjectsService.Setup(service => service.PostProject(null)).Throws(new ProjectsException(ProjectsErrors.BadRequest, new FluentValidation.ValidationException("BadRequest")));
+            var response = await this.postProject.CreateProject(request);
+            var objectResult = Assert.IsType<ObjectResult>(response);
```

Comment:
Here you can add one more Assert to validate the http error code is the right one

Reply:
Done